packages/cli/src/request.ts

```diff
@@ -3,76 +3,26 @@ import ora from 'ora';
 
 const spinner = ora();
 
-interface RequestParams {
-  [key: string]: any;
-}
+export type RequestParams = Record<string, any>;
 
-export interface ResponseParams {
-  [key: string]: any;
-}
+export type ResponseParams = Record<string, any>;
 
-export async function get(host: string, params?: RequestParams): Promise<ResponseParams> {
+const createGeneralRqeuest = (agent: Function) => async (...args: any[]) => {
   try {
-    let response = await axios.get(host, {
-      params
-    });
+    let response = await agent(...args);
     if (response.data.status === true) {
       return response.data.data;
     }
   } catch (err) {
-    if (err.response && err.response.data && err.response.data.message) {
+    if (err?.response.data.message) {
```

Comment:
```ts
err?.response.data.message // now
err.response && err.response.data && err.response.data.message // before
```
They are not exactly the same, `err?.response.data.message` didn't verify the safety of operation in `response` and `data`. This is because I consider that they are not verified too under the normal side above. 

I'm not certain if `response` and `data` need to be verified here, use `?` will bring additional code, it depends on you

Reply:
will verify safety of each operation